multinet/__init__.py

```diff
@@ -21,13 +21,48 @@
 sentry_sdk.init(dsn=sentry_dsn, integrations=[FlaskIntegration()])
 
 
-def get_allowed_origins() -> List[str]:
+def get_allowed_origins() -> Set[str]:
     """Read in comma-separated list of allowed origins from environment."""
     allowed_origins = os.getenv("ALLOWED_ORIGINS", default=None)
     if allowed_origins is None:
-        return []
+        return set()
 
-    return [s.strip() for s in allowed_origins.split(",")]
+    return {s.strip() for s in allowed_origins.split(",")}
+
+
+def register_cors_func(app: Flask, with_credentials: bool = False) -> None:
```

Comment:
I liked the form of this better that included the `allowed_origins` argument. Why not keep that?

Reply:
It's because now the allowed origins are stored in `app.config["ALLOWED_ORIGINS"]`, and are grabbed from there. This is necessary if we want testing to work properly, as otherwise the allowed origins are only set on app creation, and can't be modified. I also think that storing this option in the flask config is cleaner anyway,

Reply:
But you can still store these in the app config, and just pass them to `CORS_wildcard()` as a keyword argument, right? Instead of referencing `app.config["ALLOWED_ORIGINS"]` in line 45 below, it could just refer to the argument, and then you would pass in `allowed_origins = app.config["ALLOWED_ORIGINS"]`. Why would this interfere with testing?